src/app/modules/auth/register.strings.ts

```diff
@@ -44,3 +44,6 @@ export const OR = "Ou";
 
 export const CTA_TO_LOGIN = "Já possui uma conta?";
 export const LOGIN = "Faça o Login!";
+
+export const REGISTER_SUCCESS = "Conta criada com sucesso!";
+export const REGISTER_ERROR = "Erro ao criar conta";
```

Comment:
```suggestion
export const REGISTER_ERROR = "Erro ao criar conta.";
```

Pra manter a consistência, se você está adicionando pontuação em uma das mensagens, faz sentido adicionar nas outras também